helpers/formatNumber.ts

```diff
@@ -25,8 +25,8 @@ export const formatNumberToNScale = (number: number | string, fixed = 2, spacer
   return `${retNumber}${spacer}${unit}`;
 };
 
-export const formatNumberToCurrency = (number: number | string, options = {}) =>
-  new Intl.NumberFormat("en", options).format(Number(number));
+export const formatNumberToCurrency = (number: number | string, maximumFractionDigits = 2) =>
```

Comment:
Nope, we might want to use more options than just the maximumFractionDigits (pex, round up or down, if needed) this has to be possible; We can, however, document `options: NumberFormatOption` (or whichever the name of the interface is, and default that to `= {maximumFractionDigits: 2}`